functions/integral.go

```diff
@@ -14,6 +14,8 @@ const IntegralKind = "integral"
 
 type IntegralOpSpec struct {
 	Unit flux.Duration `json:"unit"`
+	TimeSrc string   `json:"timeSrc"`
+	TimeDst string   `json:"timeDst"`
```

Comment:
Why are we adding it to the integral function?

Reply:
the timeSrc is used for computation in the integral.  We could remove timeDst, I'm not sure if that provides context for the value or not.

Reply:
@nathanielc I made an update that I believe addresses your concerns.  We still need to identify the input timeCol because that column is required for computing the integral.  The timeDST is not needed.

Reply:
OK, that makes sense

Reply:
Do we have a time destination for derivative? The time is likely an important part of this function, but I'm not sure how we treat derivative which would be similar.